### PR TITLE
Fix correct elemental API resource version

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: docs/upgrade.md [hunk 1]
```diff
@@ -33,13 +33,13 @@ spec:
 ### Managing available versions
 
 An upgrade channel file (
-`rancheros-v0.0.0-amd64.upgradechannel-amd64.yaml` ) file is shipped
-in Elemental releases and can be applied in a Kubernetes cluster where the rancheros operator is installed to syncronize available version for upgrades.
+`elemental-v0.0.0-amd64.upgradechannel-amd64.yaml` ) file is shipped
```
Comment (Itxaka) [inline docs/upgrade.md:36]: wow, first time I read about this, Im not sure where this is generated from lol

**Review by Itxaka — APPROVED** [review]
nice!

**Review by ldevulder — APPROVED** [review]

**Review by kkaempf — APPROVED** [review]
More docs, more good

**Comment by mjura** [comment]
It was already merged in https://github.com/rancher/elemental/pull/260

**Pull request closed without merging**